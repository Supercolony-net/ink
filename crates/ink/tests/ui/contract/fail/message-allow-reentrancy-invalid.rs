#[ink::contract]
mod contract {
    #[ink(storage)]
    pub struct Contract {}

    impl Contract {
        #[ink(constructor)]
        pub fn constructor() -> Self {
            Self {}
        }

        #[ink(message, allow_reentrancy = true)]
        pub fn message(&self) {}
    }
}

fn main() {}
