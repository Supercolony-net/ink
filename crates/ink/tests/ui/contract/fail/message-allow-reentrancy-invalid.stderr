error: unknown ink! attribute argument (name = value)
  --> tests/ui/contract/fail/message-allow-reentrancy-invalid.rs:12:24
   |
12 |         #[ink(message, allow_reentrancy = true)]
   |                        ^^^^^^^^^^^^^^^^^^^^^^^
